scripts/build-android.js

```diff
@@ -81,7 +78,7 @@ for (const arch of architectures) {
     "-GNinja",
     `-DANDROID_NDK=${ndkPath}`,
     `-DANDROID_ABI=${arch}`,
-    `-DCMAKE_MAKE_PROGRAM=${sdkPath}/cmake/${cmakeVersion}/bin/ninja`,
+    `-DCMAKE_MAKE_PROGRAM=ninja`,
```

Comment:
Should we have a pre-condition check to verify that the correct version of `ninja` is going to be used?

Reply:
Probably a good idea.  Merging for now, as we have complaints.  Lets look this over afterwards.